app/views/users/_form.html.erb

```diff
@@ -46,8 +45,11 @@
     <label>Email:</label>
     <%= text_field_tag :email, current_user.email %><br>
 
+    <label>Please enter password to confirm:</label>
+    <%=text_field_tag :password, current_user.password %>
+
```

Comment:
This was genius Logan haha I loved this work around. 

I _think_ the way we could make it update without providing a password would be by adding `allow_nil: true` to the user model validation. But I haven't fully tested it out yet.

`validates :password, confirmation: true, presence: true, allow_nil: true`

Just FYI no need to change it unless you want to!